app/src/main/java/ch/epfl/culturequest/ui/profile/ProfileFragment.java

```diff
@@ -32,12 +32,19 @@ public View onCreateView(@NonNull LayoutInflater inflater,
         binding = FragmentProfileBinding.inflate(inflater, container, false);
         View root = binding.getRoot();
 
-        final TextView textView = binding.profileName;
+        final TextView profileName = binding.profileName;
+        final TextView profilePlace = binding.profilePlace;
         final CircleImageView profilePicture = binding.profilePicture;
         final RecyclerView pictureGrid = binding.pictureGrid;
 
+        FollowButton followButton = new FollowButton(binding.profileFollowButton);
+        profileViewModel.getFollowed().observe(getViewLifecycleOwner(), followButton::setFollowed);
```

Comment:
Just a question about the UI, now where is the personal Profile Fragment that only displays the Profile of the current User ?
Did you override this one for this sprint to only display the Follow button and the profile picture ?

Reply:
Waiting for Lucas that managed to separate these cases, I just added the button as is in the profile fragment for now.